algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -43,6 +43,8 @@ object HyperLogLog {
   /* Size of the hash in bits */
   val hashSize = 128
 
+  val powersOfNegativeTwo: Array[Double] = 0.until(hashSize).map{ i => math.pow(2.0, -i) }.toArray
```

Comment:
Hm I'm confused as to what's being asked here.

@ianoc I'm quite certain we don't need `hashSize-1`
@jnievelt IIRC the definition of rhoW is "index of first non-zero", not "number of zeros" so that might explain starting at 1?
@kamalmarhubi argh, good point